quickstart.sh

```
@@ -0,0 +1,171 @@
+read -p "Or enter your own task for your agent: " TASK_INPUT
+TASK_INPUT="${TASK_INPUT:-$DEFAULT_TASK}"
+
+# Interactive menu for inference model
+options=("gemini/gemini-2.0-flash" "anthropic/claude-3-5-sonnet-latest" "openai/gpt-4o")
+SELECTED_MODEL="${options[0]}"
+
+function select_option {
+
+    # little helpers for terminal print control and key input
+    ESC=$( printf "\033")
+    cursor_blink_on()  { printf "$ESC[?25h"; }
```

Shellcheck (0.10.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)